internal/client/client_test.go

```diff
@@ -21,9 +23,12 @@ import (
 // socketPath is the path to the temporary unix domain socket
 var socketPath string
 
+// socket is the client connection to the server on socketPath
+var socket *http.Client
```

Comment:
Hm, is there no way to avoid having this as a global variable and rather pass it down into the test? If that's not possible, feel free to leave as is.

Reply:
Reading the testing docs I don't see any way to pass things into the individual tests.